app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -67,6 +74,13 @@ class SelectionInteractionViewModel private constructor(
           )
         }
       }
+    if (selectedAnswer.size == 1) {
+      choiceItems[selectedAnswer[0]].handleItemClicked()
```

Comment:
This seems like a hacky way to do this. Couldn't we pass in whether a specific item is selected when creating its view model? That seems cleaner than pretending that the user clicked on the element.

Consider that future changes to handleOnClicked() may not anticipate this specific use case (that it can be called outside of a user interaction).